internal/controller/nemo_evaluator_controller.go

```diff
@@ -447,3 +460,21 @@ func (r *NemoEvaluatorReconciler) renderAndSyncResource(ctx context.Context, Nem
 	}
 	return nil
 }
+
+func (r *NemoEvaluatorReconciler) getKubernetesSecret(ctx context.Context, namespace, secretName, key string) (string, error) {
```

Comment:
nit: getValueFromSecret() better suits this function

Reply:
Fixed